<launch>
    <arg name="db_path" />
    <arg name="overwrite" default="false" />

    <!-- Run the Social Warehouse -->
    <node name="social_warehouse" type="mongo_wrapper_ros.py" pkg="social_memory" output="screen">
        <param name="host" value="localhost" />
        <param name="port" value="33402" />
        <param name="overwrite" value="$(arg overwrite)" />
        <param name="db_path" value="$(arg db_path)" />
    </node>

    <!-- Run the Social Memory -->
    <node name="social_memory" type="social_memory_node.py" pkg="social_memory" output="screen">
        <rosparam command="load" file="$(find social_memory)/config/social_events.yaml"/>
        <rosparam command="load" file="$(find social_memory)/config/system_events.yaml"/>
        <param name="event_period" type="double" value="0.5"/>
    </node>
</launch>
